src/Valet/Constants.cs

```diff
@@ -0,0 +1,42 @@
+namespace Valet;
+
+public static class Constants
+{
+    public static readonly List<string> UserInputVariables = new()
+    {
+        "GITHUB_ACCESS_TOKEN",
+        "GITHUB_INSTANCE_URL",
+        "AZURE_DEVOPS_ACCESS_TOKEN",
+        "AZURE_DEVOPS_PROJECT",
+        "AZURE_DEVOPS_ORGANIZATION",
+        "AZURE_DEVOPS_INSTANCE_URL",
+        "CIRCLE_CI_ACCESS_TOKEN",
+        "CIRCLE_CI_INSTANCE_URL",
+        "CIRCLE_CI_ORGANIZATION",
+        "CIRCLE_CI_PROVIDER",
+        "GITLAB_INSTANCE_URL",
+        "GITLAB_ACCESS_TOKEN",
+        "JENKINSFILE_ACCESS_TOKEN",
+        "JENKINS_USERNAME",
+        "JENKINS_ACCESS_TOKEN",
+        "JENKINS_INSTANCE_URL",
+        "TRAVIS_CI_ACCESS_TOKEN",
+        "TRAVIS_CI_INSTANCE_URL",
+        "TRAVIS_CI_SOURCE_GITHUB_ACCESS_TOKEN",
```

Comment:
Will this always be a copy of 'GITHUB_ACCESS_TOKEN' above?

Reply:
Not necessarily. It could be different if the GitHub repo that contains the original travis ci pipeline is in a different org/instance than is being migrated to.

That said, it most commonly will be the same value. We could use a default value of `$GITHUB_ACCESS_TOKEN` for this since the `dotenv` gem that is used internally supports [variable substitution](https://github.com/bkeepers/dotenv#variable-substitution).